packages/ubuntu_widgets/lib/src/clickable_text.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+/// This widget is a text that can be clicked and when it is hovered it changes
+/// its style, by default to underline + primary color, but that can be
+/// customized with [textHoverStyle].
+class CategoryText extends StatefulWidget {
```

Comment:
Should it be called `ClickableText` then?

Reply:
Lol, copy paste mistake, thanks!